tools/requirements.txt

```diff
@@ -1,6 +1,6 @@
-lottie_specs[docs]==0.2
+lottie_specs[docs]==0.5
 mkdocs-cinder==1.2.0
-# mkdocs-print-site-plugin==2.5.0
-git+https://github.com/mbasaglia/mkdocs-print-site-plugin.git@rename-page
+mkdocs-print-site-plugin==2.7.3
+# git+https://github.com/mbasaglia/mkdocs-print-site-plugin.git@rename-page
```

Comment:
For reference, the new mkdocs-print-site-plugin release includes the changes we pushed to it